pkg/render/node.go

```diff
@@ -968,6 +968,10 @@ func (c *nodeComponent) nodeEnvVars() []v1.EnvVar {
 		}},
 	}
 
+	if c.cr.CNI != nil && c.cr.CNI.Type == operator.PluginAmazonVPC {
+		nodeEnv = append(nodeEnv, v1.EnvVar{Name: "FELIX_BPFEXTTOSERVICECONNMARK", Value: "0x80"})
```

Comment:
Should this only been done when using BPF mode? Or maybe it doesn't matter in non-bpf modes.

Reply:
It does not matter in non-BPF modes.